dScripts/LupGenericInteract.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include "CppScripts.h"
+
+class LupGenericInteract : public CppScripts::Script {
+public:
+	void OnUse(Entity* self, Entity* user) override;
+private:
+	const uint32_t frakjawChestId = 16486;
```

Comment:
Copy and paste error?

Reply:
Yes, ty. I usually copy another script she just that as a base. Got distracted with moving platform stuff and missed this